### DISCUSSION TITLE
how to change mail container width

### DISCUSSION THREAD

**Discussion by sysmat** [shellscape/jsx-email#113] [Q&A] [answered]
- I try with `style` on `Body, Container`  not working
```tsx
const main = {
  backgroundColor: "#f6f9fc",

  fontFamily:
    '-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Ubuntu,sans-serif',
};

const container = {
  backgroundColor: "#ffffff",
  margin: "0 auto",
  marginBottom: "64px",
  padding: "20px 0 48px",
  width: "61em",
};

<Body style={main}>
      <Container style={container}>
...
``` 

- when modifying `style max-width` in generated html it works

```html
<table align="center"
           width="100%"
           data-id="@jsx-email/container"
           role="presentation"
           cellSpacing="0"
           cellPadding="0"
           border="0"
           style="max-width:60em;background-color:#ffffff;margin:0 auto;margin-bottom:64px;padding:20px 0 48px">
```

![jsx-email-starter tsx](https://github.com/shellscape/jsx-email/assets/69830/0fb91603-5199-4cbb-ac77-b455dc60a6e6)

**Comment by shellscape** [comment]
There's a `disableDefaultStyle` prop on `Container` (and a few other components) which should allow you to totally control it from a class now, or to bypass previously set default styles, like the one causing the problem here.

**Reply by lordelogos** [reply, marked as answer]
Example usage:

```tsx
<Container disableDefaultStyle style={container}>
```

**Reply by sysmat** [reply]
- jsx prop of style  `maxWidth` which translate in html style `max-width`
